File: Mission10Assignment/Models/Bowler.cs
namespace Mission10Assignment.Models
{
    public class Bowler
    {
        // grabbing all of the values
        // primary key
        public int BowlerId { get; set; }
        public string BowlerLastName { get; set; }
        public string BowlerFirstName { get; set; }
        public string? BowlerMiddleInit { get; set; }

        public string BowlerAddress { get; set; }
        public string BowlerCity { get; set; }
        public string BowlerState { get; set; }
        public string BowlerZip { get; set; }

        public string BowlerPhoneNumber { get; set; }

        // This links the Bowler to the Team in my other folder (Foreign Key)
        public int TeamId { get; set; }
        public virtual Team? Team { get; set; }
    }
}

File: Mission10Assignment/frontend/src/types/Bowler.ts
export interface Team {
  teamId: number;
  teamName: string;
}

export interface Bowler {
  bowlerId: number;
  bowlerFirstName: string;
  bowlerLastName: string;
  bowlerMiddleInit: string | null;
  bowlerAddress: string;
  bowlerCity: string;
  bowlerState: string;
  bowlerZip: string;
  bowlerPhoneNumber: string;
  teamId: number;
  team: Team | null;
}
